EST.MIT.InvoiceImporter.Function/Services/EventQueueService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Azure.Storage.Queues;
+using EST.MIT.InvoiceImporter.Function.Interfaces;
+using EST.MIT.InvoiceImporter.Function.Models;
+using InvoiceImporter.Function.Models;
+using System;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace EST.MIT.InvoiceImporter.Function.Services;
+
+public class EventQueueService : IEventQueueService
+{
+    private readonly QueueClient _queueClient;
+
+    public EventQueueService(QueueClient queueClient)
+    {
+        _queueClient = queueClient;
+    }
+
+    public async Task CreateMessage(string id, string status, string action, string message, Invoice? invoice = null)
+    {
```

Comment:
Does CreateMessage need any null checks?

Reply:
Do you mean the parameter needs to be checked against null?